src/index.js

```diff
@@ -1 +1,84 @@
+/* eslint-disable no-console */
 'use strict';
+
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+const formidable = require('formidable');
+
+const PORT = process.env.PORT || '3000';
+
+const server = http.Server();
+
+server.on('request', (req, res) => {
+  if (req.url === '/data' && req.method === 'POST') {
+    const form = new formidable.IncomingForm();
+
+    form.parse(req, (err, fields, files) => {
+      if (err) {
+        res.statusCode = 400;
+        res.end(JSON.stringify(err));
```

Comment:
It's a bad practice to send a user server errors. Consider specifying a string with general information